docs/tre-admins/auth.md

```diff
@@ -22,9 +22,9 @@ You can build all of the Identity assets by running the following at the command
 ```bash
 make auth
 ```
-This will create five identities, and if successful will write a new file; `/devops/auth.env`. If you are building locally, these values will be used when building your TRE. If you are setting this up for CI/CD, then these values will be needed by your Build Orchestrator.
+This will create five identities, and if successful will write the outputs to athentication section in `config.yaml` file. If you are building locally, these values will be used when building your TRE. If you are setting this up for CI/CD, then these values will be needed by your Build Orchestrator.
 
-The contents of your `/devops/auth.env` file should contain :
+The contents of your athentication section in `config.yaml` file should contain :
```

Comment:
typo `athentication` should be `authentication`

Reply:
Fixed, I think we should install a spellchecker in our devcontainer. Maybe create an issue for that and add it... WDYT?

Reply:
yeah...maybe....I'm not a big fan of spell checkers as you end up fighting with them a lot when spellchecking code

Reply:
You convinced me right away :)